src/scripting/lua_bases.cc

```diff
@@ -162,11 +167,12 @@ int LuaEditorGameBase::get_players(lua_State* L) {
 /* RST
    .. function:: get_immovable_description(immovable_name)
 
-      :arg immovable_name: the name of the immovable
-
-      Returns the ImmovableDescription for the named object.
+      Returns the description of the named immovable depending on the type of the immovable.
```

Comment:
What do you mean by "type"? This is only used for the derived Immovable type such as trees, rocks, wheatfields, but *not* for BaseImmovables such as buildings. an object returned by this function should always have `type_name "immovable"`.